config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/guests/:id", to: "guests#show"
+  post "/guests/:id/rooms", to: "guest_rooms#create"
```

Comment:
following through with my comment on your form earlier, we could change this route to be: 

```ruby
post "/guest_rooms", to: "guest_rooms#create"
```